readme.md

```
@@ -13,82 +13,422 @@
+| Auth            | Supabase Auth (GoTrue)         | Supabase             |
+| Realtime (chat) | Supabase Realtime              | Supabase             |
+| Videollamadas   | LiveKit Cloud                  | LiveKit              |
+| Traducción      | DeepL API                      | DeepL                |
+| Imágenes        | ImageKit                       | ImageKit             |
+| Email           | MailJet                        | MailJet              |
+| Deploy          | Docker en Railway              | Railway              |
+| Auth social     | Google OAuth                   | Google Cloud         |
+
+### **2.3. Descripción de alto nivel del proyecto y estructura de ficheros:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)